www/source/docs/reference/reference.html.md.erb

```diff
@@ -1260,13 +1260,43 @@ Packages are installed in the `/hab/pkgs/` directory, and then further organized
 ---
 ## <a name="sup-log-configuration" id="sup-log-configuration" data-magellan-target="sup-log-configuration">Supervisor Log Configuration Reference</a>
 
-With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are two main ways of configuring logging, each of which has their own strengths and weaknesses: using environment variables, and using a configuration file.
+With the 0.83.0 release of the Chef Habitat Supervisor, it is possible to have greater control over logging output, including the ability to dynamically adjust the logging configuration of a running Supervisor. There are three ways of configuring logging, each of which has their own strengths and weaknesses: as part of the `hab` command, by using environment variables, and by using a configuration file.
 
-### Environment Variable Configuration
+### RUST_LOG with a command
+
+Use `RUST_LOG=debug RUST_BACKTRACE=1` to see an individual command's debug and backtrace.
+
+    ```bash
+    # Linux/MacOS
+    # replace "hab sup run" with your command
+    RUST_LOG=debug RUST_BACKTRACE=1 hab sup run
+    ```
+
+    ```powershell
+    # Windows
+    # replace "hab sup run" with your command
+    $env:RUST_LOG="debug"; $env:RUST_BACKTRACE=1; hab sup run
+    ```
+
+### RUST_LOG as an Environment Variable Configuration
 
 It is still possible to configure logging via the `RUST_LOG` environment variable. This approach is often useful for quickly re-configuring logging (requiring a restart), or for easily configuring logging in container-based deployments.
 
-The configuration scheme is essentially the same as that described in the documentation of the Rust [env_logger](https://docs.rs/env_logger/0.6.1/env_logger/#enabling-logging) crate, with the exception that we do not allow additional regular expression-based filtering. The configuration values the Supervisor recognizes are described below:
+```bash
+export HAB_FEAT_EVENT_STREAM = “1”
+export RUST_LOG=“debug”
```

Comment:
To @mwrock point above, the difference between `VAR=foo command` and `export VAR=foo; command` is an OS concern and not Habitat specific. We can probably omit one of the formats to simplify this page. I do think having one of them is useful for people to copy/paste as this is a common question. 

---

When you run a command in Bash (and I believe this is true for all shells, but I'm not 100% certain on that), the command is run in a process with its own environment. `export` allows the specified variable and its value set in the current shell to be inherited by the spawned process.

By way of example, given the following shell script `var.sh`
```
#!/bin/bash

echo "VAR is $VAR"
```
You get the following outputs based on how `VAR` is set:
```
$ echo $VAR

$ ./var.sh
VAR is
$ VAR="hello"
$ ./var.sh
VAR is
$ echo $VAR
hello
$ VAR="hello" ./var.sh
VAR is hello
$ export VAR="hi"
$ ./var.sh
VAR is hi
$ VAR="hello" ./var.sh
VAR is hello
```